engine/language-server/src/test/scala/org/enso/languageserver/filemanager/ContentRootManagerSpec.scala

```diff
@@ -28,12 +29,20 @@ class ContentRootManagerSpec
     with Futures
     with Inside
     with EitherValue
-    with OptionValues {
+    with OptionValues
+    with BeforeAndAfterEach {
+
+  var rootManager: ContentRootManagerWrapper = _
+  var rootActor: ActorRef                    = _
+  var rootProjectDirectory: File             = _
```

Comment:
We have `org.enso.testkit.WithTemporaryDirectory` for these purposes. Please use it instead.

Reply:
done